### PR TITLE
FIT PostProc: bugfix for incorrect types in dynamic cast

### PR DESCRIPTION
**Pull request by afurs**

### CODE REVIEW

**Comment by afurs** [comment]
@Barthelemy please add this PR into next QC version

**Comment by afurs** [comment]
Unrelated CICD issue

File: Modules/FIT/FDD/src/PostProcTask.cxx [hunk 1]
```diff
@@ -306,13 +306,13 @@ void PostProcTask::update(Trigger t, framework::ServiceRegistryRef)
   }
 
   auto mo3 = mDatabase->retrieveMO(mPathDigitQcTask, "AmpPerChannel", t.timestamp, t.activity);
-  auto hAmpPerChannel = mo3 ? dynamic_cast<TH2D*>(mo3->getObject()) : nullptr;
+  auto hAmpPerChannel = mo3 ? dynamic_cast<TH2F*>(mo3->getObject()) : nullptr;
```
Comment (knopers8) [inline Modules/FIT/FDD/src/PostProcTask.cxx:309]: Why not `TH2`, so it works for both?

Comment (afurs) [reply Modules/FIT/FDD/src/PostProcTask.cxx:309]: I think better to cast into exact type of this object.

**Review by knopers8 — APPROVED** [review]

File: Modules/FIT/FDD/src/PostProcTask.cxx (hunk 1 above)
Comment (knopers8) [reply Modules/FIT/FDD/src/PostProcTask.cxx:309]: I hope we do not get to a case where QC objects are produced with a QC version with `TH2D`, but the checks are applied with a newer QC version...

**Pull request merged**